Task56/reacttask/src/App.js

```diff
@@ -0,0 +1,65 @@
+
+import React, {useState} from "react";
+import Timer from "./components/timer/Timer";
+import TimerOnRef from "./components/timer/TimerOnRef";
+import SearchInput from "./components/search_input/SearchInput";
+
+
+const SearchInputModes = {Immediate: 1, AfterPause: 2, ByEnter: 3}
+
+const App = () => {
+  const settings = null
+  const onTimerCompleted = null
+  const strings = ['123', 'abrpetr', 'abracadabra']
```

Comment:
Move it as constant outside of the component. Otherwise array would be recreated on every rerender.